pkg/verifier/notation/notation.go

```diff
@@ -220,19 +244,22 @@ func (v *notationPluginVerifier) GetNestedReferences() []string {
 }
 
 // append namespace to certStore so they are uniquely identifiable
-func prependNamespaceToCertStore(verificationCertStore map[string][]string, namespace string) (map[string][]string, error) {
+func prependNamespaceToCertStore(verificationCertStores map[string]interface{}, namespace string) (map[string]interface{}, error) {
 	if namespace == "" {
 		return nil, re.ErrorCodeEnvNotSet.WithComponentType(re.Verifier).WithDetail("failure to parse VerificationCertStores, namespace for VerificationCertStores must be provided")
 	}
-
-	for _, certStores := range verificationCertStore {
-		for i, certstore := range certStores {
-			if !isNamespacedNamed(certstore) {
-				certStores[i] = namespace + constants.NamespaceSeperator + certstore
+	for _, trustStoreType := range trustStoreTypes {
+		if mapValue, ok := verificationCertStores[trustStoreType].(map[string]interface{}); ok {
```

Comment:
what if it's not a map[string]interface{}?

Reply:
There are situations where a valid type of `trustStore` has not been init, e.g. `"signingAuthority"`